other/templates/request.j2

```diff
@@ -1,8 +1,27 @@
 
 async def {{ name.lower_snake_case }}(self{% for param in params %}, {{ param.name.lower_snake_case }}{% endfor %}):
-    """ Generated by dcsdkgen
+    """
+ {{ indent(method_description, 1) }}
+
+     {% if params -%}
+     Parameters
+     ----------
```

Comment:
According to PEP it would probably be more like this:
```
    Parameters:
```

Reply:
I followed the [numpy](https://numpydoc.readthedocs.io/en/latest/format.html) examples, I really don't mind. I just know that PyCharm understands what we have now. I will need to check what happens with `Parameters:`.

I don't find where in PEP it defines how documentation should be written. If there is something that is more standard than numpy, I'm fine changing it.

Reply:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

That's all I found. But otherwise go with what numpy suggests, looks fine.